internal/pkg/pkiinit/option/optionHelpers.go

```diff
@@ -164,3 +170,47 @@ func zeroOutFile(fileToErase string) error {
 	}
 	return nil
 }
+
+func checkIfFileExists(fileName string) bool {
+	fileInfo, statErr := os.Stat(fileName)
+	if os.IsNotExist(statErr) {
+		return false
+	}
+	return !fileInfo.IsDir()
+}
+
+func writeSentinel(sentinelFilename string) error {
+	timestamp := []byte(strconv.FormatInt(time.Now().Unix(), 10))
+	return ioutil.WriteFile(sentinelFilename, timestamp, 0400)
+}
+
+// markComplete creates sentinel files of all services to signal pki-init deploy is done
+func markComplete(dirPath string) error {
+	// recursively walk through all sub-directories until reach the leaf node
+	// to write the sentinel files
+	fileDescs, readErr := ioutil.ReadDir(dirPath)
+	if readErr != nil {
+		return readErr
+	}
+
+	for _, fileDesc := range fileDescs {
+		aFilePath := filepath.Join(dirPath, fileDesc.Name())
```

Comment:
not the biggest deal but is there a better variable name for this other than aFilePath.

Reply:
not sure what other better variable name I can think of... `aFilePath` pretty much fits its purpose here